test/text_map_codec.js

```diff
@@ -68,4 +69,24 @@ describe('Text Map Codec should', () => {
     // metrics
     assert.isOk(LocalBackend.counterEquals(metrics.tracesStartedSampled, 1));
   });
+
+  it('not set debug flag when debug-id-header is received but operation is throttled', () => {
```

Comment:
Does this mean that headers indicating the trace is a debug trace will be ignored and subsequent spans will not be marked with the debug flag?

Reply:
Yes since it's throttled.

Reply:
Sounds kind of ruthless... is this consistent with the other clients?

Reply:
this is the first place we're implementing it. How is being throttled via debug header vs throttled vs sampling.priority any different? We should be defending against both.